src/LiveDevelopment/LiveDevMultiBrowser.js

```diff
@@ -91,8 +91,8 @@ define(function (require, exports, module) {
 
     // XXXBramble: we need to get this loaded so our extension can require/use it later
     require("LiveDevelopment/Servers/BaseServer");
-    
```

Comment:
Revert all changes to this file (`git checkout master src/LiveDevelopment/LiveDevMultiBrowser.js`)